packages/geoprocessing/src/components/SketchAttributesCard.stories.tsx

```diff
@@ -25,18 +25,20 @@ export const simple = () => (
             exportId: "DESIGNATION",
             fieldType: "ChoiceField",
             label: "Designation",
-            value: "Marine Reserve"
+            value: "Marine Reserve",
           },
           {
             exportId: "COMMENTS",
             fieldType: "TextArea",
             label: "Comments",
-            value: "This is my MPA and it is going to be the greatest. Amazing."
-          }
-        ]
+            value:
+              "This is my MPA and it is going to be the greatest. Amazing.",
+          },
+        ],
       },
       geometryUri: "",
-      projectUrl: "https://example.com/project"
+      projectUrl: "https://example.com/project",
+      visibleLayers: [],
```

Comment:
What do you think about a story that renders a single layer in both checked and unchecked state? Or however you implemented the UI in your report.